pkgs/tools/networking/vpn-slice/default.nix

```diff
@@ -13,26 +13,38 @@
 buildPythonApplication rec {
   pname = "vpn-slice";
   version = "0.16.1";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "dlenski";
-    repo = pname;
+    repo = "vpn-slice";
     rev = "v${version}";
     sha256 = "sha256-T6VULLNRLWO4OcAsuTmhty6H4EhinyxQSg0dfv2DUJs=";
   };
 
   postPatch =
     lib.optionalString stdenv.hostPlatform.isDarwin ''
       substituteInPlace vpn_slice/mac.py \
-        --replace "'/sbin/route'" "'${unixtools.route}/bin/route'"
+        --replace-fail "'/sbin/route'" "'${unixtools.route}/bin/route'"
     ''
     + lib.optionalString stdenv.hostPlatform.isLinux ''
       substituteInPlace vpn_slice/linux.py \
-        --replace "'/sbin/ip'" "'${iproute2}/bin/ip'" \
-        --replace "'/sbin/iptables'" "'${iptables}/bin/iptables'"
+        --replace-fail "'/sbin/ip'" "'${iproute2}/bin/ip'" \
+        --replace-fail "'/sbin/iptables'" "'${iptables}/bin/iptables'"
     '';
 
-  propagatedBuildInputs = with python3Packages; [
+  # One would expect setuptools should be part of the build-system
```

Comment:
Please remove this entire comment. What you are experience just means that the tool also has a runtime dependency on setuptools which means we need it in both which is totally valid in the python ecosystem.

Just according to the source code, I couldn't find any reference to setuptools besides the setup.py file which is not needed at runtime. Where is that actually required? What is the error you

Reply:
[x] done